assets/javascripts/discourse/helpers/format-enabled-automation.js.es6

```diff
@@ -1,9 +1,15 @@
 import { iconHTML } from "discourse-common/lib/icon-library";
 import { registerUnbound } from "discourse-common/lib/helpers";
 
-registerUnbound("format-enabled-automation", function (enabled) {
-  return (enabled
-    ? iconHTML("check", { class: "enabled-automation" })
-    : iconHTML("times", { class: "disabled-automation" })
+registerUnbound("format-enabled-automation", function(enabled, trigger) {
+  return (enabled && trigger.id
```

Comment:
Maybe a normal `if / else` would read better here? That's quite a large ternary

Reply:
sure